src/devuan.ipxe

```diff
@@ -11,9 +11,10 @@ clear devuan_version
 clear older_release
 menu ${os} - ${arch_a} - Image Sig Checks: [${img_sigs_enabled}]
 item --gap Latest Releases
+item ascii ${space} ${os} ascii (stretch)
 item jessie ${space} ${os} 1.0 (jessie)
 item --gap Testing Releases
-item stretch ${space} ${os} ascii (testing)
+item beowulf ${space} ${os} ceres (testing)
```

Comment:
Thanks for the contrib, would something like this work well?  If so, please update the PR
```
item ascii ${space} ${os} ascii (stable)
item beowulf ${space} ${os} beowulf (testing)
item ceres ${space} ${os} ceres (unstable)
```